stan/math/prim/prob/poisson_binomial_lccdf.hpp

```diff
@@ -0,0 +1,67 @@
+#ifndef STAN_MATH_PRIM_PROB_POISSON_BINOMIAL_LCCDF_HPP
+#define STAN_MATH_PRIM_PROB_POISSON_BINOMIAL_LCCDF_HPP
+
+#include <stan/math/prim/meta.hpp>
+#include <stan/math/prim/err.hpp>
+#include <stan/math/prim/fun/binomial_coefficient_log.hpp>
+#include <stan/math/prim/fun/log.hpp>
+#include <stan/math/prim/fun/log1m.hpp>
+#include <stan/math/prim/fun/log_sum_exp.hpp>
+#include <stan/math/prim/fun/log1m_exp.hpp>
+#include <stan/math/prim/fun/max_size.hpp>
+#include <stan/math/prim/fun/multiply_log.hpp>
+#include <stan/math/prim/fun/size.hpp>
+#include <stan/math/prim/fun/size_zero.hpp>
+#include <stan/math/prim/fun/value_of.hpp>
+#include <stan/math/prim/fun/poisson_binomial_log_probs.hpp>
+
+namespace stan {
+namespace math {
+
+/** \ingroup prob_dists
+ * Returns the log CCDF for the Poisson-binomial distribution evaluated at the
+ * specified number of successes and probabilities of successes.
+ *
+ * @tparam T_y type of number of succeses parameter
+ * @tparam T_theta type of chance of success parameters
+ * @param y array of numbers of successes
+ * @param theta array of chances of success parameters
```

Comment:
After generalising the templating, these inputs can be a wider range of types. Maybe something like:

```
@param y Input scalar, vector, or nested vector of numbers of successes
```